pkg/pillar/types/kube.go

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2024 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package types
+
+const (
+	// EVEKubeNameSpace : Common namespace for all EVE kubernetes components
+	EVEKubeNameSpace = "eve-kube-app"
+	// EVEkubeConfigFile : EVE k3s config file
+	EVEkubeConfigFile = "/run/.kube/k3s/k3s.yaml"
```

Comment:
Why do you want to use a hidden directory .kube? Doesn't make debuggging any easier.

Reply:
k3s.yaml contains the secret certificates hence we wanted to keep it hidden.

Reply:
@zedi-pramodh , keep the directory hidden will not make it inaccessible, so I agree with Erik, there is no reason to hide it, please, let me know if I'm missing some understanding here...

Reply:
@rene sure I understand hidden files can be searched too. In fact we are open source and if someone wants they can see all config files we use. Only reason we thought is since its a critical config file to connect to kubernetes server, we might as well hide it and will not be accidentally deleted if someone playing on the box. But I am ok to unhide it if that helps in anyway.

Reply:
@zedi-pramodh could you please elaborate on "secret certificates", maybe we can find a better way to protect them.

Reply:
@shjala that file is generated by kubernetes when it starts and is used by clients to connect to it. Looks something like this.
apiVersion: v1
clusters:
- cluster:
    certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUJkakNDQVIyZ0F3SUJBZ0lCQURBS0JnZ3Foa2pPUFFRREFqQWpNU0V3SHdZRFZRUUREQmhyTTNNdGMyVnkKZG1WeUxXTmhRREUzTURRNU5ERXhNVGN3SGhjTk1qUXdNVEV4TURJME5URTNXaGNOTXpRd01UQTRNREkwTlRFMwpXakFqTVNFd0h3WURWUVFEREJock0zTXRjMlZ5ZG1WeUxXTmhRREUzTURRNU5ERXhNVGN3V1RBVEJnY3Foa2pPClBRSUJCZ2dxaGtqT1BRTUJCd05DQUFUdkduYlNCZTRYSzBFRjNmN2R6dnFaQmJoYTR3NGJnbmlCS1U2LzE3WmUKZmE0MVFFMzBTbXRBZ0pNWE1HNW5HdmgzMGVURlQyeXVMTzhlK284cDJsd0ZvMEl3UURBT0JnTlZIUThCQWY4RQpCQU1DQXFRd0R3WURWUjBUQVFIL0JBVXdBd0VCL3pBZEJnTlZIUTRFRmdRVVVvaXB1eHhZdzZTRzAwbmZCc1E5Cmc0TFFxUDh3Q2dZSUtvWkl6ajBFQXdJRFJ3QXdSQUlnS1dFZStEY1BadElFYjV5ck05VHdwNFhXRkFnM3h4V3YKalNpaDNQOGhyc29DSUFmUGZjdEQ2dUVLYWRDeE1oNG5hZUhLemRUN01paWRaU1NxY0wrUzNNdGEKLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=
    server: https://127.0.0.1:6443
  name: default
contexts:
- context:
    cluster: default
    user: default
  name: default
current-context: default
kind: Config
preferences: {}
users:
- name: default
  user:
    client-certificate-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUJrVENDQVRlZ0F3SUJBZ0lJSUErWTNQVUIyUW93Q2dZSUtvWkl6ajBFQXdJd0l6RWhNQjhHQTFVRUF3d1kKYXpOekxXTnNhV1Z1ZEMxallVQXhOekEwT1RReE1URTNNQjRYRFRJME1ERXhNVEF5TkRVeE4xb1hEVEkxTURFeApNREF5TkRVeE4xb3dNREVYTUJVR0ExVUVDaE1PYzNsemRHVnRPbTFoYzNSbGNuTXhGVEFUQmdOVkJBTVRESE41CmMzUmxiVHBoWkcxcGJqQlpNQk1HQnlxR1NNNDlBZ0VHQ0NxR1NNNDlBd0VIQTBJQUJJbFRHcmVhMklnTlRhMjEKUzI5d2pmdnhhKzJkYWlpQlM1YVhRYnhubXVkb2ZUZE5WVk02eXFINTkwcGRwVXRtdTFEYnQrMmlMVDE3Y2FFUwpSK1p6dGI2alNEQkdNQTRHQTFVZER3RUIvd1FFQXdJRm9EQVRCZ05WSFNVRUREQUtCZ2dyQmdFRkJRY0RBakFmCkJnTlZIU01FR0RBV2dCUkFqREJCZ0x5dG1kWGpBelU1ckE1M0hOK0IzVEFLQmdncWhrak9QUVFEQWdOSUFEQkYKQWlFQSt0ZGd2SnpqTDFnZ2VTcy9UL3g4UWVWOXhhek50V25ZZmMxd2JCQXFLNWdDSUVickZCd09uSGFwRkVmZQpVMHdjYWtZVzgvclNUaWMwYlk3b3pTNmZyRGsyCi0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0KLS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUJkekNDQVIyZ0F3SUJBZ0lCQURBS0JnZ3Foa2pPUFFRREFqQWpNU0V3SHdZRFZRUUREQmhyTTNNdFkyeHAKWlc1MExXTmhRREUzTURRNU5ERXhNVGN3SGhjTk1qUXdNVEV4TURJME5URTNXaGNOTXpRd01UQTRNREkwTlRFMwpXakFqTVNFd0h3WURWUVFEREJock0zTXRZMnhwWlc1MExXTmhRREUzTURRNU5ERXhNVGN3V1RBVEJnY3Foa2pPClBRSUJCZ2dxaGtqT1BRTUJCd05DQUFRZ0huaFM4SU5oMGhoV3hGWlNiaDFpSlJOL0tsd2xwM3VCb1MzTHVubmQKVk9jWVFCTkpDR3JoZWJHNm50RUhUellCaUtETVVONjlldTVzdnZGZkxLSkpvMEl3UURBT0JnTlZIUThCQWY4RQpCQU1DQXFRd0R3WURWUjBUQVFIL0JBVXdBd0VCL3pBZEJnTlZIUTRFRmdRVVFJd3dRWUM4clpuVjR3TTFPYXdPCmR4emZnZDB3Q2dZSUtvWkl6ajBFQXdJRFNBQXdSUUloQUkxM0ZEbzVoSDlQbVk3L1F0YXRFdzRLeGgzelZPQ3cKajlVanZHSWh1VUNSQWlCYnIxblhlYUc5T2N2aktKNWQrZjlkNE52b1U0YWMzK0FqU2hGMURmcjkxZz09Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K
    client-key-data: LS0tLS1CRUdJTiBFQyBQUklWQVRFIEtFWS0tLS0tCk1IY0NBUUVFSUcybVo1YlNZYU8vS1lUS0k2Y2RaWjMrU2xWNXJ0eG1VTy91Zy84bzNQZ2NvQW9HQ0NxR1NNNDkKQXdFSG9VUURRZ0FFaVZNYXQ1cllpQTFOcmJWTGIzQ04rL0ZyN1oxcUtJRkxscGRCdkdlYTUyaDlOMDFWVXpySwpvZm4zU2wybFMyYTdVTnUzN2FJdFBYdHhvUkpINW5PMXZnPT0KLS0tLS1FTkQgRUMgUFJJVkFURSBLRVktLS0tLQo=


But since pillar micro services are the  client here and no one from outside connects to it or can connect to it, we kept in /run